enumerables.rb

```diff
@@ -0,0 +1,143 @@
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield (self[i])
+      i += 1
+    end
+  end
+
+  def my_each_with_index
+    i = 0
+    while i < length
+      yield self[i], i
+      i += 1
+    end
+  end
+
+  def my_select
+    selected = []
+    my_each { |i| selected << i if yield(i) }
+    selected
+  end
+
+  def my_all?
+    result = true
+    my_each { |i| result = false unless yield(i) }
+    result
+  end
+
+  def my_any?
+    result = false
+    my_each { |i| result = true if yield(i) }
+    result
+  end
+
+  def my_none?
+    result = true
```

Comment:
* [ ] There should be a working `my_none?`  identical and returns the same thing as ruby's [`none?`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-none-3F). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_none.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
#### Let us compare the results return in each of the examples below and make the return the same result as the original method.
1. ```p %w{ant bear cat}none?(/d/)``` and ```p %w{ant bear cat}.my_none?(/d/)``` must return same result.
1. ```p [1, 3.14, 42].none?(Float)``` and ```p [1, 3.14, 42].my_none?(Float)``` must return same result.
1. ```p [nil].none?``` and ```p [nil].my_none?``` must return same result.
1. ```p [nil, false].none?``` and ```p [nil, false].my_none?``` must return same result.
1. ```p [nil, false, true].none?``` and ```p [nil, false, true].my_none?``` must return same result.